packages/core/src/utils/entry-points.test.ts

```diff
@@ -25,6 +27,10 @@ describe('getPackageEntryPoints', () => {
       },
       packageRoot,
     );
+
+    context.enterWith({
+      root: packageRoot,
+    } as any);
```

Comment:
Would it be better to cas to `EnhancedConfig` instead? Just in case it changes drastically and the `any` eats the error completely.
```suggestion
    } as EnhancedConfig);
```